hack/cstor_volume/test/util.sh

```diff
@@ -1,28 +1,28 @@
 #!bin/bash
 # kubectl related generic functions
-kubectlApply()
+
+prefixedFileNames()
 {
     # append all the files together prefixed with -f
-    filesToApply=""
+    local filesToApply=""
     while [ "$1" != "" ]; do
         filesToApply="$filesToApply -f $1"
         sleep 1
         shift
     done
 
+    echo $filesToApply
```

Comment:
Can you add some informative comment in echo. printing `-f abc -f ghh -f ggg` does not make any sense.

Reply:
Done with the requested changes. Please have a look.